README.md

```diff
@@ -1,7 +1,68 @@
-# Downloads folder cleaner
+# Downloads Folder Cleaner for Mac
+
+Nice and clean "Downloads" folder everyday, automatically.
+
+## Shut up and take my money
+
+No, you don't need to pay. Here is the one-line installer (for free!!!):
+
+```sh
+sh -c "$(curl -fsSL https://raw.githubusercontent.com/ascendbruce/downloads-folder-cleaner/master/install.sh)"
+```
+
+We will walk through the detail later if you don't know how to install.
+
+## Benefits
+
+* No more painful spring cleaning in Downloads folder.
+* No extra actions need to be taken.
+* Save disk space.
+
+## How it works
+
+* It moves old files into Trash once a day. (old file: added into Downloads folder more than 24 hours ago)
```

Comment:
Can just combine into one sentence, might be clearer:

`It moves old files that have been added into Downloads folder more than 24 hours ago`